auditai/cmh.py

```diff
@@ -0,0 +1,125 @@
+from scipy import stats
+from functools import partial
+import numpy as np
+import pandas as pd
+
+def extract_data(df):
+    # utility function for cmh test and odds ratio
+    # df needs to be a 2x2 data frame with columns 'pass' and 'fail'
+    a = float(df.iloc[0]['pass'])
```

Comment:
i'd generally prefer if these were less hardcoded. e.g. instead of requiring specific input columns, can include`pass_col='pass'` (assuming it's binary, after which the `fail` is understood) as an argument

Reply:
added!